### DISCUSSION TITLE
In the BTC donation address still correct?

### DISCUSSION THREAD

**Discussion by AArnott** [AvaloniaUI/Avalonia#10481] [Q&A] [answered]
I see it was Dec 2021 when the BTC donation address in the readme [was originally added](https://github.com/AvaloniaUI/Avalonia/commit/dea04d8a7455cee215860ea6ab1de063ee1b121e). Since then, sad to see that [no one has donated that way](https://www.blockchain.com/explorer/addresses/btc/bc1q05wx78qemgy9x6ytl5ljk2xrt00yqargyjm8gx). I'd love to be the first. But before I do, I just wanted to check that someone on your team still has the private key to that address. Can someone confirm?

That said, [BTC transaction fees being what they are](https://ycharts.com/indicators/bitcoin_average_transaction_fee#:~:text=Bitcoin%20Average%20Transaction%20Fee%20is%20at%20a%20current,yesterday%20and%2028.75%25%20from%20one%20year%20ago.%20Stats), I think I'd rather donate in Zcash or Nano. Is there any interest in setting up either of those? 

I'd also do BCH, LTC, ARRR, XLM. But Zcash and Nano are my favorite.

**Comment by MikeCodesDotNET** [comment, marked as answer]
Hey, 

I've just checked with the team, and the wallet address is correct and should work. The majority of people are donating via Open Collective and GitHub Sponsors, so you would be the first to donate via BTC 🥇

**Reply by AArnott** [reply]
Great. Although BTC costs ~$5 to withdraw from Binance, and that's before the network fee. So... give me a better cryptocurrency option (any that I mentioned in my OP) and I'll send some your way. Perhaps repeatedly, as I'm likely to be a regular Avalonia user now that I've converted an app over to Avalonia fully.
